README.md

```diff
@@ -196,7 +196,11 @@ _Donglai Xiang, Hanbyul Joo, Yaser Sheikh_
 [\[back to top\]](#contents)
 
 ### 2019 Others
-##### [2019 SIGGRAPH] Cascaded Point Network for 3D Hand Pose Estimation. [\[PDF\]](https://handtracker.mpi-inf.mpg.de/projects/TwoHands/content/TwoHands_SIGGRAPH2019.pdf) [\[Project\]](https://handtracker.mpi-inf.mpg.de/projects/TwoHands/)
+
+##### [2019 SIGGRAPH] InteractionFusion: Real-time Reconstruction of Hand Poses and Deformable Objects in Hand-object Interactions. [\[PDF\]](https://dl.acm.org/citation.cfm?id=3322998)
+_Hao Zhang, Zi-hao Bo, Jun-Hai Yong, Fengxu_

```

Comment:
Thanks. Please format the author names properly, Zi-Hao Bo,... Feng Xu.

Reply:
Sorry for the mistake. I fixed it in the new commit.